src/main/java/net/masterthought/cucumber/generators/EscapeHtmlReference.java

```diff
@@ -2,16 +2,27 @@
 
 import org.apache.commons.lang.StringEscapeUtils;
 import org.apache.velocity.app.event.ReferenceInsertionEventHandler;
+import org.owasp.html.HtmlPolicyBuilder;
+import org.owasp.html.PolicyFactory;
 
 /**
  * Escapes all html and xml that was provided in a reference before inserting it into a template.
+ *
+ * References that start with $_sanitize_ will be sanitized to allow urls.
  */
 final class EscapeHtmlReference implements ReferenceInsertionEventHandler {
 
+    private static final PolicyFactory LINKS = new HtmlPolicyBuilder()
+            .allowStandardUrlProtocols().allowElements("a").allowAttributes("href")
```

Comment:
what if someone wants to display the image?

I think that it is defined by Jenkins administrator so we should allows for more than white list with single element that soon or later will be extended. What is your opinion?